service/raft/group0_state_machine.hh

```diff
@@ -46,14 +46,22 @@ struct topology_change {
     std::vector<canonical_mutation> mutations;
 };
 
+struct mixed_change {
+    unsigned int topo_muts_count = 1;
+    std::vector<canonical_mutation> mutations;
+    // TODO:
+    // 1. enforce mutations order? Hardcode it?
+    // 2. Store counts of each type, not only topology?
```

Comment:
Having two vectors seems simpler than this counting stuff...

Reply:
It's not the best, but I am under the impression it has to stay that way and kind-of 'mimick' other similar structs layout, i.e. have everything in a single vector, because of what the API expects, e.g. in this method:
```
std::vector<canonical_mutation>& group0_state_machine_merger::get_command_mutations(group0_command& cmd)
```
we have to return a reference to a single vector containing all the mutations.

Reply:
Removed the `count`, left only a single vector, just like other structs.